apps/trading/components/market-banner/market-monitoring-auction.tsx

```diff
@@ -34,10 +31,23 @@ export const MarketAuctionBanner = ({
     auctionEnd && getDateTimeFormat().format(new Date(auctionEnd));
 
   const remaining = auctionEnd
-    ? new Date(auctionEnd).getTime() - getCurrentTimeInLocal().getTime()
+    ? new Date(auctionEnd).getTime() - Date.now()
     : 0;
 
-  const timeRemaining = getTimeFormat().format(new Date(remaining));
+  const formatRemaining = (remaining: number) => {
```

Comment:
**nitpick (non-blocking)**: I'd move this out to a separate function so it could be tested separately and simplify the component, but it's clear enough!